src/service-worker.js

```diff
@@ -185,8 +193,10 @@ self.addEventListener('push', event => {
 
 async function showSimpleNotification (data) {
   await self.registration.showNotification(data.title, {
+    badge: '/icon-push-badge.png',
     icon: data.icon,
     body: data.body,
+    tag: data.notification_id,
```

Comment:
If this is unrelated to this PR, could you open it as a separate PR? Thanks!

Reply:
Done #2299